tests/conftest.py

```diff
@@ -42,9 +42,6 @@ def create_yaml_workflow_schema():
         '''
         version: 0.3.0
         inputs:
-          files:
-            - code/helloworld.py
-            - inputs/names.txt
```

Comment:
Necessary?  Would be nice to keep the fixture equal to the full example...

Reply:
I added these lines back.